oat-sa/extension-tao-mediamanager
# Feature/AUT-17/Support multiple column layout in passage

Related to: https://oat-sa.atlassian.net/browse/AUT-17
Related to: https://github.com/oat-sa/extension-tao-itemqti/pull/1646

How to test:
- go to Passage Authoring
- Check that blocks can be manipulated in the same way as they can in item authoring: dropped above, left, right, below an existing block, resized in 1/12 steps, removed or via the “plus button”.

![image](https://user-images.githubusercontent.com/25976342/108960319-c653ad80-7686-11eb-8fac-0b561e2f18cf.png)
